src/discussions/common/AuthorLabel.jsx

```diff
@@ -40,7 +40,7 @@ function AuthorLabel({
   const className = classNames('d-flex align-items-center', labelColor);
 
   const showUserNameAsLink = useShowLearnersTab()
-      && linkToProfile && author && author !== messages.anonymous;
+      && linkToProfile && author && author !== 'anonymous';
```

Comment:
are you sure the author name is returned as anonymous in data?

Reply:
Yes, It will be anonymous